### PR TITLE
Add rector

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

File: tests/Model/BaseManagerTest.php [hunk 1]
```diff
@@ -46,10 +46,7 @@ final class BaseManagerTest extends TestCase
      */
     private $objectManager;
 
-    /**
-     * @var ManagerTest
-     */
-    private $manager;
+    private \Sonata\Doctrine\Tests\Model\ManagerTest $manager;
```
Comment (jordisala1991) [inline tests/Model/BaseManagerTest.php]: This needs a use.

Comment (VincentLanglet) [reply tests/Model/BaseManagerTest.php]: Fixed

**Comment by VincentLanglet** [comment]
Dev kit: https://github.com/sonata-project/dev-kit/pull/2034

File: src/Mapper/DoctrineCollector.php [hunk 2]
```diff
@@ -21,42 +21,39 @@ final class DoctrineCollector
...
      * @var array<class-string, array<string, array<array<string, mixed>>>>
      */
-    private $overrides = [];
+    private array $overrides = [];
 
-    /**
-     * @var DoctrineCollector
-     */
-    private static $instance;
+    private static ?\Sonata\Doctrine\Mapper\DoctrineCollector $instance = null;
```
Comment (jordisala1991) [inline src/Mapper/DoctrineCollector.php]: This also needs a use

Comment (VincentLanglet) [reply src/Mapper/DoctrineCollector.php]: Done

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**